src/index.jsx

```diff
@@ -12,12 +13,40 @@ import Footer, { messages as footerMessages } from '@edx/frontend-component-foot
 import appMessages from './i18n';
 import './index.scss';
 import ProgramRecordsList from './components/ProgramRecordsList';
+import ProgramRecord from './components/ProgramRecord';
 
 subscribe(APP_READY, () => {
   ReactDOM.render(
     <AppProvider>
       <Header />
-      <ProgramRecordsList />
+      {getConfig().SITE_NAME === 'localhost' ? (
```

Comment:
Can you help me understand what this does? Why are we searching for localhost? Was this intentional? Is this to hide the newer components from being rendered?

If so, I don't think I've seen it done this way before, nor do I think we should implement it this way. I think I would much prefer a feature flag so we can control how components are rendered in an environment-specific configuration file.

Let me know if I got any of this wrong!

Reply:
I see what you mean. I used that one because it was already there but creating a new one specific to this behavior makes more sense